e_commerce_project/e_commerce_app/views.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+# Create your views here.(
+
+def main(request):
```

Comment:
Could you let me know why you created this APP? @Yshah112 @rinkishivani 
For set up, you don't need to create any app. By then, according to the feature, we need to create an app related to the feature name.

Reply:
removed application  from project directory